.mergify.yml

```
@@ -0,0 +1,93 @@
+pull_request_rules:
+  - name: ask to resolve conflict
+    conditions:
+      - -merged
+      - -closed
+      - conflict
+    actions:
+        comment:
+          message: |
+            This pull request is now in conflict. Could you fix it @{{author}}? 🙏
+            To fixup this pull request, you can check out it locally. See documentation: https://help.github.com/articles/checking-out-pull-requests-locally/
+            ```
+            git fetch upstream
+            git checkout -b {{head}} upstream/{{head}}
+            git merge upstream/{{base}}
+            git push upstream {{head}}
+            ```
+  - name: backport patches to 8.0 branch
+    conditions:
+      - merged
+      - base=master
```

Should these be main? There are a few mentions of `master` in this PR.

```suggestion
      - base=main
```